README.md

```diff
@@ -46,6 +46,14 @@ If you don't have `rust` installed, we recommend installing it via [`rustup`](ht
 (If you *are* interested in having a binary release, then please let us know in the 'Discussions' area of this
 project or by filing a feature request in 'Issues'.)
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/brush.svg)](https://repology.org/project/brush/versions)
```

Comment:
I'm a fan of repology; would be great to include the badge. Could you move the badge up with the rest of the badges near the top of the README? That will fit in well.

Reply:
Sure! I have moved it with adding a blank line to fix mismatch of different badge height.

https://github.com/reubeno/brush/pull/387/commits/9345e82ea30e4dc1c803f3381712bb2563574bcf

Reply:
This is the link for other styles of badges we can choose. https://repology.org/project/brush/badges

[![Packaging status](https://repology.org/badge/vertical-allrepos/brush.svg)](https://repology.org/project/brush/versions)

[![Packaging status](https://repology.org/badge/tiny-repos/brush.svg)](https://repology.org/project/brush/versions)

[![latest packaged version(s)](https://repology.org/badge/latest-versions/brush.svg)](https://repology.org/project/brush/versions)

[![nixpkgs unstable package](https://repology.org/badge/version-for-repo/nix_unstable/brush.svg)](https://repology.org/project/brush/versions)

Reply:
Thanks for the info; the current badge looks fine for now.